Wikipedia/UI-V5/WMFHomeNearbyCell.h

```diff
@@ -2,11 +2,20 @@
 #import "WMFSaveableTitleCollectionViewCell.h"
 @import CoreLocation;
 
-@interface WMFHomeNearbyCell : WMFSaveableTitleCollectionViewCell
+@class MWKLocationSearchResult;
 
-@property (copy, nonatomic) NSString* descriptionText;
+@interface WMFHomeNearbyCell : WMFSaveableTitleCollectionViewCell
```

Comment:
Maybe we should rename this to `WMFNearbyCell` since it is used outside of the home VC now?